go.mod

```diff
@@ -101,6 +102,7 @@ require (
 	github.com/agext/levenshtein v1.2.3 // indirect
 	github.com/agnivade/levenshtein v1.0.1 // indirect
 	github.com/apparentlymart/go-textseg/v13 v13.0.0 // indirect
+	github.com/aws/aws-sdk-go v1.17.7
```

Comment:
Shouldn't we be using `github.com/aws/aws-sdk-go-v2` now? Are glue crawlers not supported there?

Reply:
`go mod tidy` fixed it